v3/generator/schemas/schemas.go

```diff
@@ -526,3 +537,22 @@ func isNullableReference(node *yaml.Node) bool {
 
 	return false
 }
+
+func isAlphanumeric(s string) bool {
+	reg := regexp.MustCompile("^[a-zA-Z0-9]+$")
+	return reg.MatchString(s)
+}
+
+func humanizeInteger(s string) string {
+	n, err := strconv.Atoi(s)
+	if err != nil {
+		return s
+	}
+
+	if n >= 0 {
+		return s
+	}
+
+	// remove the "-" with helper
+	return helpers.ToLowerCamel(fmt.Sprintf("minus%s", s))
```

Comment:
why is ToLowerCamel still needed here?

Reply:
it's dropping the char "-" to make it lower camel: "minus-1" become "minus1"